sale_order_line_sequence/views/account_move_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
**[change]** Missing license header

```xml
<?xml version="1.0" encoding="utf-8" ?>
<!--
  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-->
```

Reply:
Are licenses needed in each file? https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst only mentions the manifest.